shawarma/src/index.ts

```diff
@@ -14,366 +14,379 @@ const log = debugModule("shawarma:index");
 const errLog = debugModule("shawarma:ERROR");
 
 const rooms: MyRooms = {};
+const retryInterval = 15000;
 
 async function main() {
-  if (process.env.SENTRY_DNS) {
-    Sentry.init({
-      dsn: process.env.SENTRY_DNS,
-      enabled: !!process.env.SENTRY_DNS,
-    });
-  }
-  // start mediasoup
-  console.log("starting mediasoup");
-  let workers: {
-    worker: Worker;
-    router: Router;
-  }[];
-  try {
-    workers = await startMediasoup();
-  } catch (err) {
-    console.log(err);
-    throw err;
-  }
-  let workerIdx = 0;
+	if (process.env.SENTRY_DNS) {
```

Comment:
match OG spacing, tab width 2 spaces, should be in prettier in root

Reply:
Thanks for catching this subtle but annoying change! I'm actually a bit puzzled by this since I'm seeing a tab width of 2 spaces on vs code and prettier is on and my settings seem to check out. I'll dig more into this.

Reply:
I just found this https://github.com/github/VisualStudio/issues/1524 , which seems to explain the tab width discrepancy between vs code and `git diff <hash>`.  Adding `useTabs: false` to the prettier config fixes this for me. Would it be OK to add that property? Incidentally, this aligns with my preference for spaces over tabs, which might explain why I've never come across this before. That still doesn't explain why no other contributor on this repo has seen this issue, though. 🤷🏻